x-pack/solutions/security/test/cloud_security_posture_functional/config.agentless.ts

```diff
@@ -30,7 +30,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.fleet.agentless.api.tls.certificate=${KBN_CERT_PATH}`,
         `--xpack.fleet.agentless.api.tls.key=${KBN_KEY_PATH}`,
         `--xpack.fleet.agentless.api.tls.ca=${CA_CERT_PATH}`,
-        `--xpack.cloud.id=something-anything`,
+        `--xpack.cloud.id="ftr_fake_cloud_id:aGVsbG8uY29tOjQ0MyRFUzEyM2FiYyRrYm4xMjNhYmM="`,
+        `--xpack.cloud.base_url="https://cloud.elastic.co"`,
+        `--xpack.cloud.deployment_url="/deployments/deploymentId"`,
```

Comment:
these were added to mimic real cloud env vars, so that Fleet has the necessary info from `cloudSetup` to create the agentless-specific fleet server host for ECH

this is the same set of cloud config passed to other ftr tests (cloud, fleet, and spaces)